couchdb/source/pkg/apis/sources/register.go

```diff
@@ -17,5 +17,5 @@ limitations under the License.
 package sources
 
 const (
-	GroupName = "sources.eventing.knative.dev"
+	GroupName = "sources.knative.dev"
```

Comment:
we should probably get rid of this file and use the GroupName in `knative.dev/eventing/pkg/apis/sources`